src/strategies/frax-finance/examples.json

```diff
@@ -16,8 +19,7 @@
         "network": "1",
         "addresses": [
             "0x36A87d1E3200225f881488E4AEedF25303FebcAe",
-            "0x8a97a178408d7027355a7ef144fdf13277cea776"
-        ],
-        "snapshot": 11818851
```

Comment:
There should be a snapshot defined in the example file here

Reply:
done